test/integration/client/client_test.go

```diff
@@ -16,7 +16,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package integration
+package client
```

Comment:
I can't comment on the above file, but shouldn't package in rbac_test.go be changed to auth ?